README.md

```diff
@@ -33,23 +33,88 @@ Mapael supports all modern browsers and Internet Explorer 9+. For older versions
 *   **Resizable** Maps are easily resizable.
 *   **Zoom** Zoom and panning abilities (also on mobile devices).
 
+## Installation
+
+### Directly in your page
+
+**Note on dependencies**: [jQuery](http://jquery.com/) and [Raphael](http://raphaeljs.com) 
+(and [Mousewheel](https://github.com/jquery/jquery-mousewheel), if needed) 
+must be loaded **before** Mapael in order to work properly.
+
+**Note on maps**: map files must be loaded **after** Mapael in order to work properly.
+
+#### Using CDN
+
+Include in your project page one of these tags:
+```html
+<script src="//cdnjs.cloudflare.com/ajax/libs/jquery-mapael/2.1.0/js/jquery.mapael.js"></script>
+<script src="//cdn.jsdelivr.net/npm/jquery-mapael@2.1.0/js/jquery.mapael.min.js"></script>
```

Comment:
Just wondering : why did you use an url from couldflare for the regular JS version, and an url from jsdelivr for the minified one ?

Reply:
Typo, the minify version should be used :)